python/gpusim_search.py

```diff
@@ -17,7 +17,9 @@ def main():
         similarity_cutoff = 0
 
         fp_binary = smiles_to_fingerprint_bin(smiles)
-        fp_qba = QtCore.QByteArray(fp_binary)
+        #fp_qba = QtCore.QByteArray(fp_binary)
```

Comment:
Let's keep the original fp_qba creation line, but unpack the results of `smiles_to_fingerprint_bin` when we call the function.